From 9b75615360f4de36f534f00943c09c605cdd8ec8 Mon Sep 17 00:00:00 2001
From: "ping.zhang" <ping.zhang@ck-telecom.com>
Date: Fri, 26 Sep 2014 18:46:37 +0800
Subject: [PATCH] [Feature][USB] (2/N)open usb serial port.

[id]:66550

[Type]
  Feature

[Projects]
  hicam-baseline

[Solution Description]
  modified:   build/target/product/common.mk

[Other info]

Change-Id: Ie3cca2f22cfd1cc1e1aed20105b3af03f4bfb072
---
 build/target/product/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/target/product/common.mk b/build/target/product/common.mk
index ff959f2..c5492d6 100755
--- a/build/target/product/common.mk
+++ b/build/target/product/common.mk
@@ -2217,6 +2217,6 @@ ifeq ($(strip $(MTK_SENSOR_SUPPORT)),no)
 endif
 
 # CKT zhangping.20140924 for add UsbACM module to compile system (feature66550)
-ifeq ($(strip $(CKT_OPEN_USB_ACM)), yes)
+ifeq ($(strip $(CKT_OPEN_USB_ACM)),yes)
     PRODUCT_PACKAGES += UsbACM
 endif
-- 
1.9.1

